apps/web/components/apps/App.tsx

```diff
@@ -56,6 +56,7 @@ const Component = ({
   descriptionItems,
   isTemplate,
   dependencies,
+  concurrentMeetings,
```

Comment:
New property in app metadata to enable concurrent meetings for conferencing apps